src/main/java/uk/gov/pay/api/model/search/ISearchPagination.java

```diff
@@ -1,12 +0,0 @@
-package uk.gov.pay.api.model.search;
-
-
-public interface ISearchPagination<T> {
```

Comment:
Looks like in other Java repos we do not use `I` for interface prefixes:

- https://github.com/alphagov/pay-adminusers/search?q=public+interface&unscoped_q=public+interface
- https://github.com/alphagov/pay-direct-debit-connector/search?p=1&q=public+interface&unscoped_q=public+interface
- https://github.com/alphagov/pay-connector/search?q=public+interface&unscoped_q=public+interface

So, I guess we want to rename that and not use this prefix as you did.

You can check with other team members.

Reply:
I thought I renamed that

Reply:
Yes one of the changes is renaming the interface

Reply:
Yes, I wanted to clarify why you renamed it:
it's because on other projects we do not use `I` prefixes for interfaces and if you want to clarify why we introduced this naming with the person who did it.
Sorry it was not very clear

Reply:
Yes, it was introduced by mistake and I am renaming it to a more consistent pattern 👍